fiftyone/utils/labelstudio.py

```diff
@@ -725,6 +732,30 @@ def _to_detection(label):
     }
     return result, ls_type, label.id
 
+def _to_instance(label, size):
+    ls_type = "brushlabels"
+    if isinstance(label, list):
+        return (
+            [_to_instance(l, size)[0] for l in label],
+            ls_type,
+            [l.id for l in label],
+        )
+
+    if isinstance(label, fol.Detections):
+        return (
+            [_to_instance(l, size)[0] for l in label.detections],
+            ls_type,
+            [l.id for l in label.detections],
+        )
+
+    brush = fou.lazy_import(
+    "label_studio_converter.brush",
+    callback=lambda: fou.ensure_import("label_studio_converter.brush"),
+    )
+    rle = brush.mask2rle(label.to_segmentation(frame_size=size).get_mask())
```

Comment:
Can you please install the [dev dependencies](https://github.com/voxel51/fiftyone/blob/develop/CONTRIBUTING.md#developer-guide)? Basically just run:

```
bash install.bash -d
```

This will install pre-commit hooks that will automatically run some code formatting tools when you commit your edits.